### PR TITLE
feat(event-bus): implement simple eventbus class

### PR DESCRIPTION
**Pull request by ilyamore88**
Task #22 

# Changelog
- Add EventBus class
- EventBus extends EventTarget methods
- Add base events for blocks

# ToDo
- Add types for event objects
- Connect the Document model to EventBus

### CODE REVIEW

**Review by gohabereg — APPROVED** [review]

File: src/utils/EventBus/EventBus.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import EventEmitter from 'eventemitter3';
```
Comment (neSpecc) [inline src/utils/EventBus/EventBus.ts:1]: why not to use native `EventTarget`?

Comment (neSpecc) [reply src/utils/EventBus/EventBus.ts:1]: 70+ kb is too much for such a simple thing, I guess.

Comment (ilyamore88) [reply src/utils/EventBus/EventBus.ts]: Replaced with EventTarget

File: src/utils/EventBus/types/EventType.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/**
+ * Event types enum that can be emitted by the document model
+ */
+export enum EventType {
+  /**
+   * The document model has been changed
+   */
+  CHANGED = 'changed',
```
Comment (gohabereg) [inline src/utils/EventBus/types/EventType.ts:8]: Please use PascalCase

**Review by gohabereg — APPROVED** [review]

**Review by neSpecc — APPROVED** [review]

**Pull request merged**